### DISCUSSION TITLE
Web Service Sender - response HEADER values

### DISCUSSION THREAD

**Discussion by githubeht** [nextgenhealthcare/connect#5814] [General]
Hi,
I'm using a Web Service Sender connector (Mirth 3.6.0 or higher) to call a SOAP web service which correctly returns the SOAP envelope with the response.
Problem: I can't get the HEADERS parameter values ​​returned by the web service call.
I tried to use GetHTTP Response Header of category HTTP Sender Functions $('responseHeaders.get').get('Header-Name')
in the Destination Response Transformer, with 'Header-Name' equal to the name of the individual HEADERS parameters returned by the web service, but this returns an empty string.
The call to the same web service made with SoapUI correctly displays the values ​​of the HEADER parameters returned by the web service.
Has anyone solved this problem?

Thanks

**Comment by pacmano1** [comment]
`$('responseHeaders').getHeader('content-type')` does work on an http sender like you mentioned and fails on a web service sender.

I would switch your web service senders to use http senders instaead,  i.e. just call the SOAP endpoint construcing the call in an http sender which is straightforward.

**Reply by githubeht** [reply]
ok, it works.
Thanks
